types/context.go

```diff
@@ -48,7 +48,7 @@ type Request = Context
 
 // Read-only accessors
 func (c Context) Context() context.Context                   { return c.baseCtx }
-func (c Context) MultiStore() storetypes.MultiStore          { return c.ms }
+func (c Context) MultiStore() storetypes.CacheMultiStore     { return c.ms }
```

Comment:
This is a breaking change

Reply:
yes, it seems I haven't added changelog. Add api breaking change log entries, and list the api changes.

Reply:
MultiStore very clearly communicates the intent of requesting a MultiStore type. If a caller wants to request a CacheMultiStore, it can call a separate CacheMultiStore method. Why does this PR change this existing method, rather than providing a new method?

Reply:
make sense, changed the related places to `CacheMultiStore`.